src/main.zig

```diff
@@ -2374,48 +2385,62 @@ fn buildOutputType(
                         fatal("unable to parse minor image version '{s}': {s}", .{ minor, @errorName(err) });
                     };
                     have_version = true;
-                } else if (mem.eql(u8, arg, "-e") or mem.eql(u8, arg, "--entry")) {
+                } else if (mem.eql(u8, arg, "-e") or mem.eql(u8, arg, "--entry") or mem.eql(u8, arg, "-entry")) {
```

Comment:
I think the issue https://github.com/MichalStrehovsky/PublishAotCross/blob/45d90a19c46ba2ce7fd6158babb045d665a678a2/src/clang.cmd#L19 is about not having a space between -e and its value; i.e. clang and gcc set shared lib's entry point to null with `-e0x0`  (which is inspired by the practice of .so files in llvm-toolchain package on linux).